src/current/_data/releases.yml

```diff
@@ -7621,4 +7621,143 @@
     docker_arm: true
     docker_arm_experimental: false
     docker_arm_limited_access: false
-  source: true
\ No newline at end of file
+  source: true
+
+- release_name: v24.3.2
+  major_version: v24.3
+  release_date: '2024-12-26'
+  release_type: Production
+  go_version: go1.22.8
+  sha: 2c5477e5ba1a433059b1d36bb725a6a89c840092
+  has_sql_only: true
+  has_sha256sum: true
+  mac:
+    mac_arm: true
+    mac_arm_experimental: true
+    mac_arm_limited_access: false
+  windows: true
+  linux:
+    linux_arm: true
+    linux_arm_experimental: false
+    linux_arm_limited_access: false
+    linux_intel_fips: true
+    linux_arm_fips: false
+  docker:
+    docker_image: cockroachdb/cockroach
+    docker_arm: true
+    docker_arm_experimental: false
+    docker_arm_limited_access: false
+  source: true
+  previous_release: v24.3.1
+  cloud_only: true
```

Comment:
let's remove these parameters for all patches:
- cloud_only
- cloud_only_message_short
- cloud_only_message

![image](https://github.com/user-attachments/assets/879d100c-e9c5-426e-b737-0ea07ac06000)